src/containers/pages/JurisdictionAssignment/helpers/RiskLabel/index.tsx

```diff
@@ -0,0 +1,22 @@
+/** shows the risk label */
+import React from 'react';
+import { TreeNode } from '../../../../../store/ducks/opensrp/hierarchies/types';
+
+interface Props {
+  metadata: any[];
+  node: TreeNode;
+}
+
+export const RiskLabel = (props: Props) => {
```

Comment:
What is this component supposed to be doing?

Reply:
added module description